lib/cosmos/gui/dialogs/legal_dialog.rb

```diff
@@ -37,17 +34,25 @@ def initialize
       label.setFrameStyle(Qt::Frame::Box)
       layout.addWidget(label)
 
+      version = Qt::Label.new("Version: " + COSMOS_VERSION)
+      version.setFont(Cosmos.getFont("Arial", 14))
+      version_layout = Qt::HBoxLayout.new
+      version_layout.addStretch
+      version_layout.addWidget(version)
+      version_layout.addStretch
+      layout.addLayout(version_layout)
+
       legal_text = ''
       legal_text_filename = File.join(::Cosmos::USERPATH, 'config', 'data', 'legal.txt')
       legal_text_filename = File.join(::Cosmos::PATH, 'data', 'legal.txt') unless File.exist?(legal_text_filename)
-      File.open(legal_text_filename, "r") {|file| legal_text << file.read}
+      legal_text << File.read(legal_text_filename)
+      legal_text.strip!
       legal_text.gsub!("\r", '') unless Kernel.is_windows?
 
-      text_edit = Qt::TextEdit.new
-      text_edit.text = legal_text
-      text_edit.setReadOnly(true)
-      text_edit.setFrameStyle(Qt::Frame::Box)
-      layout.addWidget(text_edit)
+      legal_label = Qt::Label.new(legal_text)
```

Comment:
I don't think a label will create a scroll bar for long legal texts.  That's why I used TextEdit originally.